bigtable/filter_samples.rb

```diff
@@ -0,0 +1,217 @@
+# frozen_string_literal: true
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Import google bigtable client lib
+require "google/cloud/bigtable"
+
+# [START bigtable_filters_limit_row_sample]
+def filter_limit_row_sample project_id, instance_id, table_id
+  filter = Google::Cloud::Bigtable::RowFilter.sample 0.75
+  read_with_filter project_id, instance_id, table_id, filter
+end
```

Comment:
can you move the `START` to after the `def` and the `END` before the `end`

Reply:
I can, but then the indentation will look off in the documentation

Reply:
It should be fine if the region tags are also indented. https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/f3ef01233f51b2df642126bacff69268e69101e7/vision/detect_crop_hints.rb#L18 & https://cloud.google.com/vision/docs/detecting-crop-hints#vision_crop_hint_detection-ruby

Reply:
Wow awesome, didn't know it works like that.